### ISSUE TITLE
FE | Auth-Profile | Direct Deposit Consistently Displayed

### ISSUE THREAD

**Issue by SKasimow** [department-of-veterans-affairs/va.gov-team#40723]
### Background

We would like to begin displaying the Direct deposit feature consistently so that whether or not a Veteran is receiving direct deposit payments for disability compensation, pension benefits or education benefits the direct deposit link and page will be viewable.

The [direct deposit content page](https://www.va.gov/change-direct-deposit/) has 2 alerts that need to be updated:

1. When unauthenticated, the sign in prompt still tells users they can use MHV or DS Logon to manage direct deposit online, even though they can't. We need to update the sign in prompt so it directs users to ID.me or Login.gov

2. Once authenticated, MHV users w/out 2FA are still seeing an alert prompting them to set up 2FA to change DD info online, but they can't do this online. We need to update this alert to be something similar to what we show in profile.

**Comment by mdewey** [comment]
Hey @Samara-Strauss, this change is in staging at 

https://staging.va.gov/change-direct-deposit/#how-can-i-change-my-va-direct-

Let me know if you see anything! 

cc: @andaleliz @SKasimow

**Comment by Samara-Strauss** [comment]
Hi! The first alert looks good; the second alert looks mostly good, but I noticed the text that should link to ID.me or Login.gov isn't linked up:

![image](https://user-images.githubusercontent.com/34068740/167497571-372b28be-e292-490c-8d51-2fd5e3908f05.png)

**Comment by mdewey** [comment]
Good Catch! The links are updated! 

I am sending a PR now to deploy the fix. 

![Screen Shot 2022-05-10 at 10 11 22 AM](https://user-images.githubusercontent.com/1793923/167649189-84c869d2-adbf-443c-8bb9-261c98ea77d8.png)

**Comment by Samara-Strauss** [comment]
Thanks, @mdewey! I validated that the login.gov link works, but the id.me link is redirecting back to the content page (/change-direct-deposit) instead of going to ID.me.

Also, I should have realized this yesterday, but you can remove that CTA for adding 2FA since it's no longer relevant:

![image](https://user-images.githubusercontent.com/34068740/167670554-381aecec-ff79-45a4-b21f-ca5c2a067f68.png)

**Comment by mdewey** [comment]
@Samara-Strauss The button is no longer in staging!

**Comment by Samara-Strauss** [comment]
Great! I am having trouble signing in to staging with an MHV user right now to validate this, so I'll have to try later. I'll let you know when I have a chance to vaildate.

**Comment by Samara-Strauss** [comment]
I was finally able to validate! The button is gone, and the login.gov link works, but now the ID.me link just reloads the page. Can you look into what's going on?

<img width="497" alt="image" src="https://user-images.githubusercontent.com/34068740/168669466-3ca7f046-7cd0-42b9-a5b1-4a76861f796d.png">

**Comment by mdewey** [comment]
Hmmm.. I am investigating that and I have called in Platform to help

https://dsva.slack.com/archives/CBU0KDSB1/p1652819513610839

This seems to be a bigger issue than this change. I am still looking

**Comment by Samara-Strauss** [comment]
Thank you! And sorry it's a weird issue.

**Comment by mdewey** [comment]
This is being sent to another team to work through the bug in the IAM server. 

https://dsva.slack.com/archives/CSFV4QTKN/p1652899219350979?thread_ts=1652820532.805409&cid=CSFV4QTKN

Unblocked when: The IAM team resolves the issue

**Comment by mdewey** [comment]
Sent a follow-up message on the thread linked above.

**Comment by mdewey** [comment]
I have tested this out. The fix is in staging. 

@Samara-Strauss  This is good to look at again in staging. I am awaiting word on when the fix will be in production. 

All my communications with the identity team is on the [thread](https://dsva.slack.com/archives/CSFV4QTKN/p1652899219350979?thread_ts=1652820532.805409&cid=CSFV4QTKN) linked above.

**Comment by SKasimow** [comment]
Hi @Samara-Strauss - have you had a chance to validate the fix?

**Comment by Samara-Strauss** [comment]
Hi all! Just checked this out. It looks like the fix is working as expected. When I click the login.gov link, I'm taken there to sign in. When I click the ID.me link, I'm taking to the ID.me sign in page. So, I think we are good to go to launch this whenever you're ready.

**Comment by SKasimow** [comment]
Awesome - thank you!

**Comment by Samara-Strauss** [comment]
@mdewey can we launch this today?? :D

**Comment by mdewey** [comment]
@Samara-Strauss Yes we can, 100%?

**Comment by Samara-Strauss** [comment]
Yep! 100% please. No need to phase this since the updates in the profile are live to 100% of folks already.

**Comment by mdewey** [comment]
<img width="813" alt="Screen Shot 2022-06-15 at 1 10 13 PM" src="https://user-images.githubusercontent.com/1793923/173885870-9f28188b-9ae0-41d8-8d44-1a0097306480.png">

Deployed
